### PR TITLE
Add kdump support for COS

### PR DESCRIPTION
**Pull request by xueweiz**
This pull request is to add kdump support in COS, which is the third commit in this series.
The first two is necessary as the context, and are being reviewed at #1419

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?

### CODE REVIEW

**Comment by xueweiz** [comment]
Hi sosreport maintainers, I saw the above ci pipeline failed because of pylint issues:

2.31s$ pycodestyle sos
sos/sosreport.py:1310:13: E117 over-indented (comment)
sos/sosreport.py:1311:13: E117 over-indented
sos/sosreport.py:1358:21: E117 over-indented
sos/utilities.py:120:17: E117 over-indented
sos/utilities.py:122:17: E117 over-indented
sos/plugins/__init__.py:1048:13: E117 over-indented
sos/plugins/filesys.py:54:17: E117 over-indented
The command "pycodestyle sos" exited with 1.

But seeing from the source code after the merge [1], none of these pylint warning was due to the code changes in these commits. Should I make a separate pull request to fix them? Or should I just fix them in my changes? Thanks!

[1] https://github.com/sosreport/sos/commit/3d60967099a6e3c77ab461de3a177703aa935912

**Comment by bmr-cymru** [comment]
Best guess is that the `pycodestyle` version on the Travis builders has been updated. We see problems like this from time to time (`pycodestyle` 2.3.1 on my system does not report these errors).

We will fix this up in master (the original commits introduce the problems) and you can rebase your work on top of that.

**Comment by bmr-cymru** [comment]
It may be useful to add `pycodestyle --version` to our Travis jobs to make this easier to diagnose.

File: sos/plugins/kdump.py [hunk 1]
```diff
@@ -45,4 +46,25 @@ def setup(self):
...
+        ("latest_dump", "enable capture for latest kernel crash dump",
+            "", False),
+    ]
+
+    def setup(self):
+        super(CosKDump, self).setup()
+        self.add_cmd_output('ls -alRh /var/kdump*')
+        if self.get_option("all_dumps"):
+            self.add_copy_spec([
+                "/var/kdump-*"
```
Comment (pmoravec) [inline sos/plugins/kdump.py]: Assuming `/var/kdump` is the current dump and `/var/kdump-*` are elder ones, shouldnt we collect here `/var/kdump*`?

In the current implementation, `all_dumps` will skip collection of the current dump.

**Comment by pmoravec** [comment]
Apart of the one code comment, there are multiple `pycodestyle` issues, see https://travis-ci.org/sosreport/sos/jobs/486203504 .

PR needs (trivial) rebasing to resolve merge conflicts.

**Comment by TurboTurtle** [comment]
Closing this as outdated due to COS support already being in sos via the other patchsets, and (current) kdump support being enabled there. If there are further collections desired for COS, please open a new PR rebased on current `master`

**Pull request closed without merging**